tests/expected_bindings.ml

```diff
@@ -12,6 +12,10 @@ external fn_three_parameters : Car.t -> int -> int -> Car.t = "fn_three_paramete
 external fn_four_parameters : Car.t -> int -> int -> int -> Car.t = "fn_four_parameters"
 external fn_five_parameters : Car.t -> int -> int -> int -> int -> Car.t = "fn_five_parameters"
 external fn_six_parameters : Car.t -> int -> int -> int -> int -> int -> Car.t = "fn_six_parameters_bytecode" "fn_six_parameters"
+external test_add_i32 : int32 -> int32 -> int32 = "test_add_i32"
+external test_add_usize : int -> int -> int = "test_add_usize"
```

Comment:
i guess theres no usize or unsigned int to use?

Reply:
See https://github.com/o1-labs/ocaml-gen/pull/14/commits/138f525cb6526669c956bbc53a3f93f723e3bc5f